qdb/compat.py

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright 2015 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+from __future__ import print_function
+
+import sys
+
+
+try:
+    reduce = reduce
+    PY2 = True
+except NameError:
+    from functools import reduce
+    PY2 = False
+
+
+PY3 = not PY2
+
+gevent = sys.modules.get('gevent')
```

Comment:
Oh. I thought that I only had one of these. I think I want to use the import method.